src/abyssal_tome/constants.py

```
@@ -0,0 +1,99 @@
+from pathlib import Path
+import re
+
+# --- Project Root ---
+# Assuming constants.py is in src/abyssal_tome/
+PROJECT_ROOT = Path(__file__).resolve().parent.parent.parent
+
+# --- Directories ---
+ASSETS_DIR = PROJECT_ROOT / "assets"
+FAQS_DIR = PROJECT_ROOT / "faqs"
+SCHEMAS_DIR = ASSETS_DIR / "schemas"
+INDEX_DIR = PROJECT_ROOT / "indexdir" # For Whoosh index
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)